# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/nfs-utils-getrpcbynumber_r.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- nfs-utils-1.1.0/support/nfs/svc_socket.c~	2007-05-10 20:40:57.000000000 -0700
+++ nfs-utils-1.1.0/support/nfs/svc_socket.c	2007-06-07 15:37:39.000000000 -0700
@@ -67,8 +67,10 @@
   memset (&addr, 0, sizeof (addr));
   addr.sin_family = AF_INET;
 
-  ret = getrpcbynumber_r (number, &rpcbuf, rpcdata, sizeof rpcdata,
-			  &rpcp);
+//  ret = getrpcbynumber_r (number, &rpcbuf, rpcdata, sizeof rpcdata,
+//			  &rpcp);
+  rpcp = getrpcbynumber (number);
+  ret = 0;
   if (ret == 0 && rpcp != NULL)
     {
       /* First try name.  */
